code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -107,6 +111,11 @@
 	anchored = 0
 	buckle_movable = 1
 
+/obj/structure/bed/chair/office/dismantle()
+	for(var/i = 1 to 4)
```

Comment:
This should probably reference the recipe datum to get the required amount of materials to drop.